README.md

```diff
@@ -4,54 +4,88 @@ A CLI to co-host websites published to [IPFS].
 🔌 Using local ipfs daemon via http api
-🔍 Finding DNSLinks for 3 domains
-🔗 ipfs.io         QmXZz6vQTMiu6UyGxVgpLB6xJdHvvUbhdWagJQNnxXAjpn 11.5 MB
-🔗 docs.ipfs.io    QmXrsvjeZeH6rCzgQSJycKq9fFqNgkptTqYRexzaNy4wx3 6.59 MB
-🔗 awesome.ipfs.io QmdgXaAryZpe3vQcHyhzk5kowbgxvY3XaGTYvYAhoBrHLm 7.45 MB
-📦 Total size 25.6 MB for 3 domains
-📍 Pinned ipfs.io
-📍 Pinned docs.ipfs.io
-📍 Pinned awesome.ipfs.io
+📍 docs.ipfs.io    QmNrbogjGZWgUSrbmHXydwc5b51oJQsBfHA2RkNRVf2ikc 6.86 MB
+📍 awesome.ipfs.io QmPHrA6RT2j7bEcBrdeV2z6ZWrchDguw9wBchHPr6VZcFS 7.6 MB
+📍 ipfs.io         QmYb3dbymigAPcaiQUgnPxbwgMKjvxVBrrxSvFHwbZLVkq 10 MB
+📦 Total size 24.5 MB for 3 domains
 🤝 Co-hosting 3 domains via IPFS.
 ```
 
-Passing the `--no-pin` flag you can find the total size but skip pinning it.
+Passing the `--silent` options will prevent any logging.
 
 ```console
-$ ipfs-cohost dist.ipfs.io tr.wikipedia-on-ipfs.org --no-pin
+$ ipfs-cohost ipfs.io --silent
+```
+
+### Remove cohosted website
+
+You can remove one or more websites at once:
+
+```console
+$ ipfs-cohost rm ipfs.io docs.ipfs.io awesome.ipfs.io
 🔌 Using local ipfs daemon via http api
-🔍 Finding DNSLinks for 2 domains
-🔗 dist.ipfs.io             QmVJL1ew9ytqZGR7Tg121tHEXPwbYVNxzHked3QzVgWEzD 10.6 GB
-🔗 tr.wikipedia-on-ipfs.org QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX 14.9 GB
-📦 Total size 25.5 GB for 2 domains
+✔  ipfs.io no longer cohosted.
+✔  docs.ipfs.io no longer cohosted.
+✔  awesome.ipfs.io no longer cohosted.
 ```
 
-Passing the `--silent` options will prevent any logging.
+### List cohosted websites and snapshots
+
+Use `ls` with no arguments to list the cohosted domains:
 
 ```console
-$ ipfs-cohost ipfs.io --silent
+$ ipfs-cohost ls
+🔌 Using local ipfs daemon via http api
+📍 Lazily cohosted domains:
+      docs.ipfs.io
+      ipfs.io
+📍 Fully cohosted domains:
+      arewedistributedyet.com
+      awesome.ipfs.io
+      ipfs.io
+```
+
+Use `ls` with domains as arguments to list the snapshots for each domain:
+
+```console
+$ ipfs-cohost awesome.ipfs.io
+🔌 Using local ipfs daemon via http api
+⏱ Full snapshots for ipfs.io:
+      2019-10-06_095057
+      2019-10-05_135342
 ```
 
-### Could you do this with a few lines of bash?
+### Sync the current domains
 
-Yes. That is how this command started its life. You do not need `ipfs-cohost` to co-hosts websites; the `ipfs` command can do it all!
+Check if you have the most up to date version of each website and updates the snapshots.
 
 ```console
-$ ipfs object stat /ipns/docs.ipfs.io | grep CumulativeSize
-CumulativeSize: 6591536
+$ ipfs-cohost sync
+🔌 Using local ipfs daemon via http api
+✔  Snapshots synced!
+```
+
+### Prune
 
-$ ipfs pin add /ipns/docs.ipfs.io
-pinned QmXrsvjeZeH6rCzgQSJycKq9fFqNgkptTqYRexzaNy4wx3 recursively
+Delete all snapshots but the last `n`. If `n` is not provided, all snapshots will be deleted.
+
+```console
+$ ipfs-cohost prune [n]
+🔌 Using local ipfs daemon via http api
+✔  Cohosted websites cleaned!
```

Comment:
Would be nice to know what was pruned and how much space was recovered.

Reply:
> Would be nice to know what was pruned

Can be added!

> how much space was recovered

Unless someone calls `ipfs repo gc`, no space is, in fact, recovered, but that's not the job of `ipfs-cohost`. We should not `repo gc` something without user consent I believe